.github/workflows/ci-cd.yml

```
@@ -0,0 +1,527 @@
+          TEST_DB_PORT: 5432
+          TEST_DB_NAME: certaintyai_test
+          TEST_DB_USER: postgres
+          TEST_DB_PASSWORD: postgres
+          TEST_REDIS_HOST: localhost
+          TEST_REDIS_PORT: 6379
+
+      - name: Generate test coverage
+        run: npm run test:coverage
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)